api/src/test/cards.test.ts

```diff
@@ -22,4 +29,121 @@ describe('Testing Cards', () => {
+             * Post method creates post request to specified path
+             * Send method sends the cardPostData to the post request
+             */
+            const response = await request(app.getServer())
+                .post(`${cardsRoute.path}`)
+                .send(cardData1); 
+                
+            expect(response.body.data.name).toBe(cardData1.name);
+            expect(response.body.data.description).toBe(cardData1.description);
+            expect(response.body.data.salePrice).toBe(cardData1.salePrice);
+            expect(response.body.data.marketPrice).toBe(cardData1.marketPrice);
+            expect(response.body.data.rating).toStrictEqual(cardData1.rating);
+            expect(response.body.data.image).toBe(cardData1.image);
+
+            // TODO: Test priceHistory field of response
+            // expect(response.body.data.priceHistory[0]).toBe(cardData1.priceHistory._id);
+
+            debugger; 
+
+        });
+
+        it('Create Charmander Card', async () => {
+            const cardsRoute = new CardsRoute(); 
+            const app = new App([cardsRoute]); 
+
+            const priceHistoryData2: CreatePriceHistoryDto[] = [{
+                date: new Date(2023, 6, 13), 
+                quantity: 1, 
+                price: 1
+            }]
+    
+            const priceHistoryPromises2 = priceHistoryData2.map(async (history) => {
+                const priceHistory = new PriceHistoryModel(history);
+                await priceHistory.save();
+                return priceHistory;
+              });
+              
+            const priceHistories2 = await Promise.all(priceHistoryPromises2);
+    
+            const cardData2: CreateCardDto = {
+                name: 'Charmander',
+                description: 'Fire Type',
+                salePrice: 23.10,
+                marketPrice: 46.20,
+                rating: [1, 9, 8, 2, 10],
+                image: 'Charmander.png',
+                priceHistory: priceHistories2,
+            }
+
+            /**
+             * Request method creates new HTTP request that's used to send requests
+             * Post method creates post request to specified path
+             * Send method sends the cardPostData to the post request
+             */
+            const response = await request(app.getServer())
+                .post(`${cardsRoute.path}`)
+                .send(cardData2); 
+                
+            expect(response.body.data.name).toBe(cardData2.name);
+            expect(response.body.data.description).toBe(cardData2.description);
+            expect(response.body.data.salePrice).toBe(cardData2.salePrice);
+            expect(response.body.data.marketPrice).toBe(cardData2.marketPrice);
+            expect(response.body.data.rating).toStrictEqual(cardData2.rating);
+            expect(response.body.data.image).toBe(cardData2.image);
+
+            debugger; 
```

Comment:
There are a few debugger statements left, you can take these out but I am accepting the pull request to keep us moving through the project